nutri-track-bot/src/main/java/com/nutritrack/bot/LambdaHandler.java

```diff
@@ -67,6 +68,9 @@ public void handleRequest(InputStream input, OutputStream output, Context contex
         try {
             update = updateParser.parse(new String(input.readAllBytes()));
 
+            String languageCode = TelegramFactory.getLanguageCode(update);
```

Comment:
You can use update.getLocale().getLanguage();

After `update = updateParser.parse(new String(input.readAllBytes()));` the language code is presented in the update object.